src/redux/Dragon/Dragon.js

```diff
@@ -16,7 +15,7 @@ export const fetchDragons = createAsyncThunk(
       id: dragon.id,
       name: dragon.name,
       type: dragon.type,
-      flickr_images: dragon.flickr_images,
+      img: dragon.flickr_images[0],
```

Comment:
Great job @mrsamsharma  🤝

- [ ] There seems to be an error with the way axionr is imported. I think you need to install axion in `Dragon.js`.
![error](https://user-images.githubusercontent.com/44798044/208915001-71f415e1-0855-40f7-a1d6-78a4ae6a78bc.png)